### PR TITLE
Home page changes

### PR DESCRIPTION
**Pull request by qam6227**
If a user isn't a logged in, the "get started" button will direct them to the login page. If they are logged in, it will direct them to the registration page

### CODE REVIEW

File: api/server.js [hunk 1]
```diff
@@ -44,7 +44,7 @@
      });
  });
  
- const PORT = process.env.PORT || 5000; // Setting the port number for the server
+ const PORT = process.env.PORT || 5001; // Setting the port number for the server
```
Comment (heathermoses) [inline api/server.js]: Why is the port number changed?

File: api/server_BACKUP_42885.js
Comment (heathermoses) [inline api/server_BACKUP_42885.js:1]: Can we remove this from the PR? I don't think it needs to be committed

File: api/server_BASE_42885.js
Comment (heathermoses) [inline api/server_BASE_42885.js:1]: Can we remove this from the PR? I don't think it needs to be committed

File: api/server_LOCAL_42885.js
Comment (heathermoses) [inline api/server_LOCAL_42885.js:1]: Can we remove this from the PR? I don't think it needs to be committed

File: api/server_REMOTE_42885.js
Comment (heathermoses) [inline api/server_REMOTE_42885.js:1]: Can we remove this from the PR? I don't think it needs to be committed

File: client/src/pages/Home.js [hunk 2]
```diff
@@ -6,11 +6,36 @@
...
+             navigate('/login');
+
+        }
+    }
+
+
+
+
+
+
```
Comment (heathermoses) [inline client/src/pages/Home.js]: nit: extra space

File: config/config.js
Comment (heathermoses) [inline config/config.js:1]: This definitely should not be part of this PR. We might need to erase this file somehow.

**Review by heathermoses — CHANGES_REQUESTED** [review]
Need to remove some files from this PR

File: api/server.js (hunk 1 above)
Comment (qam6227) [reply api/server.js]: Something was wrong with my port 5000 when testing. Changed it to 5001 and it worked flawlessly, I can change this back to 5000

File: config/config.js
Comment (qam6227) [reply config/config.js:1]: Just made two more commits. Please lmk if this still shows up

**Review by heathermoses — APPROVED** [review]
Looks great!
